.travis.yml

```diff
@@ -1,4 +1,6 @@
 osx_image: xcode9.4
 language: objective-c
 script:
-- "./ios/travis-ci/build-ipa.sh"
+- "./ios/travis-ci/build-ipa.sh > build_output.txt"
```

Comment:
@paweldomas, you only redirected stdout. How about stderr where errors are supposed to be anyway? They'll go where they went before this PR and now on failure they'll still be printed but then the 500 of stdout will be printed as well? I guess that's ok.

Reply:
I suppose those would not be redirected and should end up in the Travis logs

Reply:
Ok, I didn't get your question at first, but yes you're right. And I guess that is ok. If we see too many of such then there's clearly something wrong.